notebooks/book2/03/lkj_numpyro.ipynb

```diff
@@ -0,0 +1,555 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # We illustrate LKJ prior as discussed in fig 14.3 of
</pre><p>You can use <code>%capture</code> to hide this installation logs</p><p>Dr. Kevin won't like this extra logs. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/58/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>